### DISCUSSION TITLE
cant assign Internet device to network

### DISCUSSION THREAD

**Discussion by masterwishx** [netalertx/NetAlertX#168] [Q&A]
Having Problem Assign device Internet to Network

When internet parent is Internet then having error and the map is gone ,and what should be Internet parent ?

![image](https://user-images.githubusercontent.com/28630321/216757274-cf20e2ad-abbe-491e-8b4c-daa935533077.png)

![image](https://user-images.githubusercontent.com/28630321/216757320-b8905eea-0f48-463c-9a1b-1eaf517bcc79.png)

![image](https://user-images.githubusercontent.com/28630321/216757176-7b091aff-740c-42b0-89df-725e61f80894.png)

**Comment by jokob-sk** [comment]
Hey,

please follow this guide: https://github.com/jokob-sk/Pi.Alert/blob/main/docs/NETWORK_TREE.md

**Comment by masterwishx** [comment]
Yes i follow but no Internet setup is fopr parent and i cant unnasign Internet after his parent Internet

**Comment by jokob-sk** [comment]
You have to delete the device and re-run the scan then that should fix it

**Reply by masterwishx** [reply]
Thanks the problem Solved

**Comment by masterwishx** [comment]
OK thanks i will try
